enhancements/kube-apiserver/audit-policy.md

```diff
@@ -0,0 +1,428 @@
+---
+title: Audit Policy Configuration
+authors:
+  - "@sttts"
+reviewers:
+  - "@mfojtik"
+  - "@soltysh"
+  - "@deads2k"
+  - "@stlaz"
+approvers:
+  - "@mfojtik"
+  - "@mcurry"
+creation-date: 2020-03-24
+last-updated: 2020-05-14
+status: provisional
+see-also:
+replaces:
+superseded-by:
+---
+
+# Audit Policy Configuration
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions [optional]
+
+## Summary
+
+This enhancement describes a high-level API in the `config.openshift.io/v1` API group to configure the audit policy for the
+API servers in the system. The audit configuration will be part of the [`APIServers`](https://github.com/openshift/api/blob/master/config/v1/0000_10_config-operator_01_apiserver.crd.yaml) resource. 
+It applies to all API servers at once.
```

Comment:
also aggregated non-core API servers?

Reply:
No. The apiserver config.openshift.io support is not in k8s.io/apiserver. We cannot enforce it. This is in line with other settings there like proxies or TLS ciphers.